output.tf

```
@@ -1,3 +1,7 @@
 output "suggest-endpoint" {
   value = "${aws_elasticsearch_domain.suggest.endpoint}"
```

Maybe?. 🤷‍♂ Copied the output to the python app as outputted (without the https scheme) as I assumed `requests` auto resolves it but guess not.
```suggestion
  value = "https://${aws_elasticsearch_domain.suggest.endpoint}"
```

---

im confused

---

Copied the suggest endpoint output given by terraform as the `LOOKUP_URL` without the https part as I assumed the request library still resolved it but guess not.